src/core/prelude/test-env/gestures.ts

```diff
@@ -182,6 +213,19 @@ export default class Gestures {
 			this.steps.push(newPoint);
 		});
 	}
+
+	/**
+	 * Resolves provided element
```

Comment:
Returns a DOM node if the passed element is a DOM node or performs a `querySelector` to find a DOM node based on the passed string.